spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/RouteLocator.java

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Dave Syer
  */
-public interface RouteLocator {
+public interface RouteLocator extends RouteMatcher {
```

Comment:
I'm not sure if this is quite what I was looking for. I'll have to noodle on it.